src/sass/file-preview-card/file-preview-card.scss

```diff
@@ -0,0 +1,113 @@
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.file-preview-card {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-template-areas: " . . menu-action " " . center-action . " " label . . ";
+    min-width: 120px;
+    max-width: 396px;
+    box-sizing: border-box;
+    background-color: var(--color-background-secondary, #f7f7f7);
+    border-radius: var(--border-radius-100, 16px);
+    aspect-ratio: 1/1;
```

Comment:
https://caniuse.com/mdn-css_properties_aspect-ratio

Support for Safari from 15+ on, so this may be an issue with some users that are on 13-14 which we still support. The worst case is the card will not maintain the aspect ratio and the a image that has a heighter > width will extend the height of the card

Reply:
As we discussed this, it should be fine to leave it as is.